lib/src/render/table/table_col_border.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:appflowy_editor/src/render/table/table_node.dart';
+import 'package:provider/provider.dart';
+
+class TableColBorder extends StatefulWidget {
+  const TableColBorder({
+    Key? key,
+    required this.tableNode,
+    required this.colIdx,
+    required this.resizable,
+  }) : super(key: key);
+
+  final bool resizable;
+  final int colIdx;
+  final TableNode tableNode;
+
+  @override
+  State<TableColBorder> createState() => _TableColBorderState();
+}
+
+class _TableColBorderState extends State<TableColBorder> {
+  final GlobalKey _borderKey = GlobalKey();
+  bool _borderHovering = false;
+  bool _borderDragging = false;
+
+  @override
+  Widget build(BuildContext context) {
+    if (widget.resizable) {
+      return buildResizableBorder(context);
+    }
+    return buildFixedBorder(context);
+  }
+
+  MouseRegion buildResizableBorder(BuildContext context) {
+    return MouseRegion(
+      cursor: SystemMouseCursors.resizeLeftRight,
+      onEnter: (_) => setState(() => _borderHovering = true),
+      onExit: (_) => setState(() => _borderHovering = false),
+      child: GestureDetector(
+        onHorizontalDragStart: (_) => setState(() => _borderDragging = true),
+        onHorizontalDragEnd: (_) => setState(() => _borderDragging = false),
+        onHorizontalDragUpdate: (DragUpdateDetails details) {
+          RenderBox box =
+              _borderKey.currentContext?.findRenderObject() as RenderBox;
+          Offset pos = box.localToGlobal(Offset.zero);
+          double colsHeight = widget.tableNode.colsHeight;
```

Comment:
```suggestion
          final Offset pos = box.localToGlobal(Offset.zero);
          final double colsHeight = widget.tableNode.colsHeight;
```
These are not reassigned either as far as I can tell